heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -163,10 +161,20 @@ local function process_json(msg, json, parsed)
         clientId = parsed.clientId
         update_field(msg.Fields, "clientId"  , clientId)
 
-        -- restructure the main ping message
-        if parsed.type == "main" then
```

Comment:
This will now attempt to restructure *all* new-style telemetry pings (with a top-level `version` field).  Is that intended (and safe)? Notably, this change will split the `saved-session` pings too.